doc/availability.md

```diff
@@ -109,17 +109,17 @@ Reference Area UK has now become available, this is because if the user were to
 
 ### Temporal coverage
 
-For DSDs that have a time dimension, temporal coverage can be included in the DataConstraint's `ReferencePeriod` element. The `ReferencePeriod` is used to indicate the earliest and latest observation dates available for the sub-cube of data based on the current data query.  
+For DSDs that have a time dimension, temporal coverage can be included in the AvailabilityConstraint's `validFrom` and `validTo` property that are part of the Cube Region's Key Value. If provided these properties indicate the earliest and latest observation dates available for specific Component values based on the current data query.  
 
 This information may not be included in the response if the service does not have access to this information.
 
 ### Metrics
 
-The `DataConstraint` may define up to two additional Annotations used to capture the metrics for the **number of series** and the **number of observations** which will be returned if the data query presented to this API were run against the data API.  
+The `AvailabilityConstraint` may provide details on the  **number of series** and the **number of observations** which will be returned if the data query presented to this API were run against the data API.  
```

Comment:
I think we can remove one blank space between "the" and "**number"

Reply:
I think the same applies to line 46 above, before "These distinct values" and before "The meaning of the distinct values".